### ISSUE TITLE
Enable ability for bold/underline/etc for question descriptions

### ISSUE THREAD

**Issue by neel-a-mehta** [civiform/civiform#888]
In #886 we discuss the importance of linkifying URLs in question descriptions to make it easier for program admins to link to external sources, such as maps of daycare centers or tables of income criteria.

Instead of doing a hacky RegEx-y solution, instead we can let Civiform admins write questions in Markdown or using a rich-text HTML editor ([or both](https://simplemde.com/)). This may increase complexity on the Civiform admin side, but it would enable a lot more creative control and let them convey more information.

**Comment by neel-a-mehta** [comment]
#979 is an example of something we'd like to do with the markdown editor too -- adding images.

**Comment by glorialiou** [comment]
Note: Rochester has mentioned this has something they would like.

**Comment by msprenke** [comment]
Needs PRD

**Comment by elisekalstad** [comment]
Would like to add hyperlinks to custom messages (statuses AND success message)

**Comment by reemaka** [comment]
Just confirming, this is specifically for question text and help text, is that correct? i.e. the fields here:

![image](https://github.com/civiform/civiform/assets/7339672/0ed80be7-cf8c-4733-ab73-486b6524d5ba)
(screenshot is for date picker question, but same fields apply to all question types)

**Comment by msprenke** [comment]
Yes Question & help and we'd like to add custom confirmation message text

**Comment by reemaka** [comment]
The custom confirmation message is for the program, correct?

**Comment by elisekalstad** [comment]
> The custom confirmation message is for the program, correct?

Yes! The custom confirmation message is specific to each program. We'd like to be able to add formatting to that message. Right now it's hard to parse out a URL or give emphasis to specific parts of the message.

**Comment by reemaka** [comment]
Makes sense, thanks for clarifying!

**Comment by reemaka** [comment]
I think a tricky part here is how to integrate this with translations. We'll need to think about how to handle that.

**Comment by msprenke** [comment]
For translations the text formatting should be the same as for the main question.  Allow the user of bold/underline
